main.go

```diff
@@ -11,10 +11,6 @@ import (
 )
 
 func main() {
-	RunServer()
```

Comment:
My initial thought for keeping this as an external function was to split the tasks of the main method into the first part, which is running the server (real backend), and the second part, which would be the web client...

Ig for now it is fine to keep it all in `main()` though